CONTRIBUTING.md

```diff
@@ -1,19 +1,29 @@
 # Contributing
 
-At Medic Mobile we welcome and appreciate community contributions.
+The Community Health Toolkit is powered by people like you. Your contributions help us create open source technology for a new model of healthcare that reaches everyone.
 
-## Ways to contribute
+## [Ways to contribute](#ways-to-contribute)
```

Comment:
It looks weird that this is a link and a title. Can we make a new title, eg "Contents", and make "ways to contribute" a list item?

Reply:
I've actually removed it altogether! Listing Improving Our Documentation first (since this is in medic-docs) allows both options to be on a single screen without scrolling, so it became unnecessary.